<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoBox</class>
 <widget class="QMainWindow" name="InfoBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>169</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>331</width>
    <height>169</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>331</width>
    <height>169</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ET InfoBox</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>311</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="info_TXT">
       <property name="text">
        <string>Select any transform node or component and press the buttons</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="worldspace_BOX">
         <property name="text">
          <string>world space</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="selection_BOX">
         <property name="text">
          <string>print selection</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="list_BOX">
         <property name="text">
          <string>append to  lists</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="translation_BTN">
         <property name="text">
          <string>Translation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rotation_BTN">
         <property name="text">
          <string>Rotation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="scale_BTN">
         <property name="text">
          <string>Scale</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="resultInfo_TXT">
       <property name="text">
        <string>Results will be shown in the script editor, due to specifications! </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
